BaristaSwift/Common/Common/Base/Store.swift

```diff
@@ -54,7 +53,7 @@ public class Store<objectType: StoreProtocol> {
             do {
                 try store.registerSoup(soupName, withIndexSpecs: indexSpecs, error: ())
             } catch let error as NSError {
-                SalesforceSwiftLogger.log(type(of:self), level:.error, message: "\(soupName) failed to register soup: \(error.localizedDescription)")
+                SalesforceSwiftLogger.log(type(of:self), level:.error, message: "\(objectType.objectName) failed to register soup: \(error.localizedDescription)")
```

Comment:
We happen to name the soup after the object type's name but I think this code is more readable that way.